Refit/Attributes.cs

```diff
@@ -148,4 +148,19 @@ public AuthorizeAttribute(string scheme = "Bearer")
         {
         }
     }
+
+    public abstract class UnsuccessfulResponseFilterAttribute : Attribute
+    {
+        public int Order { get; set; }
+        public abstract void Filter(UnsuccessfulResponseFilterContext ctx);
+    }
+
+    public class NullOn404 : UnsuccessfulResponseFilterAttribute
+    {
+        public override void Filter(UnsuccessfulResponseFilterContext ctx)
+        {
+            if (ctx.HttpResponse.StatusCode == System.Net.HttpStatusCode.NotFound)
```

Comment:
Add `using System.Net` instead of fully qualifying here.

Reply:
Will do.